bugsnag-plugin-android-ndk/src/main/assets/include/bugsnag.h

```diff
@@ -36,6 +39,11 @@ void bugsnag_set_user_env(JNIEnv *env, char* id, char* email, char* name);
 void bugsnag_leave_breadcrumb(char *message, bsg_breadcrumb_t type);
 void bugsnag_leave_breadcrumb_env(JNIEnv *env, char *message, bsg_breadcrumb_t type);
 
+void bugsnag_add_on_error(on_error on_error);
+void bugsnag_remove_on_error(on_error on_error);
```

Comment:
Is there a reason why this can't/shouldn't follow the pattern of declaring an `_env` version of each function that does the work and a version with this parameter that uses the global pointer (and validates that Bugsnag has been initialised)?

Currently the `env` parameter in the implementation of `bugsnag_add/remove_on_error_env` isn't used.

Reply:
I have added method variants which also have an `_env` parameter